chart/templates/terraform_controller.yaml

```
@@ -35,4 +35,20 @@ spec:
               value: {{ .Values.gitImage}}
             - name: GITHUB_BLOCKED
               value: {{ .Values.githubBlocked }}
+            {{ if .Values.resourceLimitsCPU }}
```

```suggestion
            {{ if .Values.resources.limits.cpu }}
```
Will it be better to follow the style like https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#resource-requests-and-limits-of-pod-and-container?

---

correct, that is good